Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM node:boron
 
 # Update and install packages
 RUN apt-get update && \
-    apt-get install -y python python-dev python-pip python-virtualenv zip rsync && rm -rf /var/lib/apt/lists/*
+    apt-get install -y python python-dev python-pip python-virtualenv zip rsync default-jdk && \
```

Comment:
And also, lets stick to `node:boron` and not ubuntu. `node:boron` is based off debian (jessie) which is much leaner than ubuntu and would result in a smaller docker image size without a lot of unneeded bulk that would be coming with ubuntu.  

cc @sch00lb0y @imujjwal96